genienlp/predict.py

```diff
@@ -490,14 +490,23 @@ def run(args, device):
             if args.translate_return_raw_outputs:
                 with open(raw_prediction_file_name, 'w' + ('' if args.overwrite else '+')) as prediction_file:
                     for i in range(len(generation_output.example_ids)):
-                        line = (
-                            generation_output.example_ids[i]
-                            + '\t'
-                            + '\t'.join(generation_output.raw_predictions[i])
-                            + '\t'
-                            + generation_output.answers[i]
-                        )  # all outputs separated by '\t'
-                        prediction_file.write(line + '\n')
+                        if args.one_output_per_line:
```

Comment:
Thanks. I'll remove this branching thing too. We should add a function for creating the correct output.